features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/Scenario.kt

```diff
@@ -29,26 +30,33 @@ abstract class Scenario(
     protected val eventMetadata: String?
 ) : Application.ActivityLifecycleCallbacks {
 
+    /**
+     * Tracks whether the scenario is starting Bugsnag only, or running the scenario.
+     */
+    protected var startBugsnagOnly: Boolean = false
```

Comment:
This property doesn't seem to be read anywhere, so could be refactored to just a parameter.

Reply:
It's currently being passed through to `getZeroEventsLogMessages` below, but perhaps that will be overtaken by the comment there...

Reply:
Yeah, having refactored `getInterceptedLogMessages` for another comment it's now used.